src/commponents/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo } from 'react';
+import cn from 'classnames';
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+interface LoadingTodo {
```

Comment:
I've seen this type before. Extract it to a separate file and export it.
Import it wherever u need it